meterfs-migrate/migrate.c

```diff
@@ -0,0 +1,673 @@
+/*
+ * Phoenix-RTOS
+ *
+ * meterfs migration tool
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * %LICENSE%
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define MAX_FILES         64
+#define SIZE_SECTOR       4096
+#define HGRAIN            32 /* Must be able divide sector size */
```

Comment:
Language:
`Must be able to divide sector size`?
`Must divide sector size`?
`Must be a divisor of SIZE_SECTOR`?

Reply:
Done